assets/src/components/parts/janus-Carousel/Carousel.css

```diff
@@ -1,3 +1,12 @@
+/*------------------------------------------
+    Janus Image Carousel
+-----------------------------------------*/
+@import 'swiper/css';
```

Comment:
Previously, the swiper related CSS files were imported in janus_image_carousel_delivery.css files but we no longer generate this file during build process. Due to that the Carousel component was not working correctly.